MAINTAINERS.md

```diff
@@ -199,38 +199,49 @@ This [repository](https://github.com/powsybl/powsybl-metrix) also provides a C++
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)
+**Reviewers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [EtienneLt](https://github.com/EtienneLt), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)  
+**Committers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
+
+### powsybl-network-store
+This [repository](https://github.com/powsybl/powsybl-network-store-server) provides a client for the persistent implementation of the IIDM grid model.
+
+**Reviewers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [EtienneLt](https://github.com/EtienneLt), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
+**Committers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
 
 ### powsybl-network-conversion-server
 This [repository](https://github.com/powsybl/powsybl-network-conversion-server) provides a web service that exposes conversion functions.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)    
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)  
+**Reviewers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
+**Committers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
 
 ### powsybl-single-line-diagram-server
 This [repository](https://github.com/powsybl/powsybl-single-line-diagram-server) provides a web service to generate single line diagrams.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)    
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst)
+**Reviewers:** [SlimaneAmar](https://github.com/SlimaneAmar), [antoinebhs](https://github.com/antoinebhs)
+**Committers:** [SlimaneAmar](https://github.com/SlimaneAmar), [antoinebhs](https://github.com/antoinebhs)
 
 ### powsybl-diagram-viewer
 This [repository](https://github.com/powsybl/powsybl-diagram-viewer) provides a service to display network-area digrams and single-line diagrams.
 
-**Reviewers:** [flo-dup](https://github.com/flo-dup)
-**Committers:** [flo-dup](https://github.com/flo-dup)
+**Reviewers:** [SlimaneAmar](https://github.com/SlimaneAmar), [antoinebhs](https://github.com/antoinebhs)
```

Comment:
@flo-dup should be left as a reviewer and a committer. He works on a regular basis with this repository.